cypress/pages/Registpage.js

```diff
@@ -0,0 +1,153 @@
+export const Registpage = {
+    TXT_FIRSTNAME: "#firstName",
+    TXT_LASTNAME: "#lastName",
+    TXT_EMAIL: "#userEmail",
+    RDO_GENDER: "#gender-radio-2",
+    TXT_MOBILE: "#userNumber",
+    DDL_DOB: "#dateOfBirthInput",
+    DDL_DOB_MONTH: ".react-datepicker__month-select",
+    DDL_DOB_YEAR: ".react-datepicker__year-select",
+    DDL_DOB_DAY: ".react-datepicker__day--014",
+    TXT_SUBJECTS: ".subjects-auto-complete__value-container",
+    CHK_HOBBIES: "#hobbies-checkbox-2",
+    BTN_PICTURE: "#uploadPicture",
+    TXT_ADDRESS: "#currentAddress",
+    DDL_STATE: "#state input",
+    DDL_CITY: "#city input", 
+    BTN_SUBMIT: "#submit",
+    BTN_CLOSE_FINISH: "#closeLargeModal",
+
+
+    typeFirstname(firstName) {
+        cy.get(this.TXT_FIRSTNAME).type(firstName);
+        return this;
+    },
+
+    typeLastname(lastName) {
+        cy.get(this.TXT_LASTNAME).type(lastName);
+        return this;
+    },
+
+    typeEmail(email) {
+        cy.get(this.TXT_EMAIL).type(email);
+        return this;
+    },
+
+    selectGender(gender) {
+        cy.get(this.RDO_GENDER).click({force: true});
```

Comment:
Are all the force options necessary? Force is a double-edge sword, it may skip some UI bugs that prevent user to interact with elements, be careful with it